LondonBritishLibrary/orient/BLorient12513.xml

```diff
@@ -0,0 +1,614 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient12513" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>The Commentary of John Chrysostom on the Epistle to the Hebrews</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 12513</idno>
+                        <altIdentifier><idno>Or. 12513</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 59</idno></altIdentifier>
+                    </msIdentifier>
+                            <msContents>
+                                <msItem xml:id="ms_i1">
+                                    <locus from="1ra" to="3va"/> 
+                                    <title ref="LIT3171Commen"/>
```

Comment:
LIT3171Commen on just 5 pages, followed by LIT3171Commen? is item 1 an intro? an extract? a different CAe ID? a separate fragmentary msPart?

Reply:
You are right. It ist LIT3171Commen#Vita, which must be nested in LIT3171Commen.